javascript/components/FormInput.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ContactField extends Component {
```

Comment:
The name of the class should match the filename.

Reply:
Thanks, fixed!